render_delegate/light.cpp

```diff
@@ -184,14 +184,14 @@ AtString getLightType(HdSceneDelegate* delegate, const SdfPath& id)
 }
 
 auto spotLightSync = [](AtNode* light, AtNode** filter, const AtNodeEntry* nentry, const SdfPath& id,
-                        HdSceneDelegate* delegate) {
```

Comment:
Please use camelCase instead of snake_case when naming parameters. See https://github.com/Autodesk/arnold-usd/blob/master/docs/conventions.md